src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="calendar calendar--start-day-sun calendar--month-length-29">
```

Comment:
<img width="838" alt="image" src="https://user-images.githubusercontent.com/102411130/229713929-1d0666d9-afa0-47c5-a6bc-6d6fc6e3035e.png">

```suggestion
    <main class="calendar calendar--start-day--sun calendar--month-length--29">
```